bin/dune_init.ml

```diff
@@ -519,7 +519,43 @@ module Component = struct
             [ File.make_text (Path.parent_exn opam_file) (Path.basename opam_file) "" ]
           | Esy -> [ File.make_text dir "package.json" "" ]
         in
-        { dir; files = dune_project_file dir opts :: package_files }
+        let default_files =
+          let fmt_version =
+            try
+              let ic = Unix.open_process_in "ocamlformat --version" in
+              let version = input_line ic in
+              let _ = Unix.close_process_in ic in
+              (let open Pp.O in
+              Console.print_user_message
+                (User_message.make
+                   [ Pp.tag User_message.Style.Hint (Pp.verbatim "Format")
+                     ++ Pp.textf ": using ocamlversion number %s" version
+                   ]));
+              version
+            with
+            | _ ->
+              (let open Pp.O in
+              Console.print_user_message
+                (User_message.make
+                   [ Pp.tag User_message.Style.Warning (Pp.verbatim "Format")
+                     ++ Pp.textf ": couldn't get the ocamlformat version number."
+                   ]);
+              Console.print_user_message
+                (User_message.make
+                   [ Pp.tag User_message.Style.Hint (Pp.verbatim "Format")
+                     ++ Pp.textf ": Are you sure you have ocamlformat installed ?"
+                   ]));
+              ""
```

Comment:
I'm not sure what to do here, but:

- I don't think we should have a catch-all exception handler, lest we hide unrelated errors
- If I have chosen not to use ocamlformat as part of my workflow, then I don't want this noise every time I initialize a project.